src/main/java/org/cascadebot/cascadebot/commands/core/StatusCommand.java

```diff
@@ -0,0 +1,51 @@
+package org.cascadebot.cascadebot.commands.core;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.JDA;
+import net.dv8tion.jda.core.JDAInfo;
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandCore;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+
+public class StatusCommand implements ICommandCore {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        EmbedBuilder builder = MessagingObjects.getClearThreadLocalEmbedBuilder();
+        JDA jda = CascadeBot.INS.getShardManager().getShardById(context.getChannel().getJDA().getShardInfo().getShardId());
+
+        builder.setTitle(context.getSelfUser().getName());
+        builder.setThumbnail(context.getSelfUser().getAvatarUrl());
+
+        builder.addField("Cascade Version", CascadeBot.getVersion().toString(), true);
+        builder.addField("JDA Version", JDAInfo.VERSION, true);
```

Comment:
Add versions for Lavaplayer maybe? Idk if they have it

Reply:
I can't find anything that might give a Lavaplayer version, sozzles.